packages/starlight/translations/tr.json

```
@@ -18,11 +18,11 @@
   "page.lastUpdated": "Son güncelleme:",
   "page.previousLink": "Önceki",
   "page.nextLink": "Sonraki",
-  "page.draft": "This content is a draft and will not be included in production builds.",
-  "404.text": "Sayfa bulunamadı. URL'i kontrol edin ya da arama çubuğunu kullanmayı deneyin.",
-  "aside.note": "Note",
-  "aside.tip": "Tip",
-  "aside.caution": "Caution",
-  "aside.danger": "Danger",
-  "fileTree.directory": "Directory"
+  "page.draft": "Bu içerik taslaktır ve canlı sürümde bulunmayacaktır.",
+  "404.text": "Sayfa bulunamadı. Bağlantıyı kontrol edin ya da arama çubuğunu kullanmayı deneyin.",
```

I’m guessing this should also maybe not change, and reflect the English text which uses “URL” rather than “link”?

```suggestion
  "404.text": "Sayfa bulunamadı. URL'i kontrol edin ya da arama çubuğunu kullanmayı deneyin.",
```

---

We can keep as suggested change from @delucis. "URL" is used in Turkish as abbreviation, too. "Bağlantı" refers to "link" word so they have different meanings.